### PR TITLE
Ci speed up

### PR DESCRIPTION
**Pull request by IagoLast**
This PR changes the travis CI configuration to use yarn instead npm

This results in faster builds.
[my first build took 8m](https://travis-ci.org/CartoDB/deep-insights.js/builds/261810116?utm_source=github_status&utm_medium=notification)
screen shot 2017-08-07 at 15 43 05

while [the build for this PR](https://travis-ci.org/CartoDB/deep-insights.js/builds/261832242) took 4m
screen shot 2017-08-07 at 15 44 05

Notice that future builds will be even faster due the yarn cache.

(Edit): [final build time](https://travis-ci.org/CartoDB/deep-insights.js/builds/261835591?utm_source=github_status&utm_medium=notification): 2m

screen shot 2017-08-07 at 15 48 40

### CODE REVIEW

**Comment by xavijam** [comment]
F****** master.

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,11 @@
-cache: false
 language: node_js
 node_js:
-  - "4.1"
-
-install:
-  - npm install
-
+  - "6"
```
Comment (xavijam) [inline .travis.yml]: I'd specify 6.9.2, just in case, because it is the version we are used to use.

Comment (IagoLast) [inline .travis.yml]: fixed

**Comment by xavijam** [comment]
dale!

**Pull request merged**